src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+
+import ProfilePage from './pages/ProfilePage';
+import TrendingPage from './pages/TrendingPage';
+import AppBar from './components/MenuAppBar';
```

Comment:
why you use here different names for the file's name and the component's name(AppBar and MenuAppBar)? With this name, changing is difficult to find exact components.